src/main/java/mat/client/cqlworkspace/shared/CQLEditorPanel.java

```diff
@@ -29,7 +29,6 @@ public CQLEditorPanel(String name, String text, boolean isReadOnly) {
 		this.isReadOnly = isReadOnly;
 		this.editor = new CQLEditor(isReadOnly);
 		this.editor.getElement().getElementsByTagName("textarea").getItem(0).setTitle(text);

```

Comment:
Does this need an aria label or something for screenreaders?

Reply:
The parent tag already contains an aria-label
aria-label="You are entering a CQL Editor. To exit the editor backward press shift and up arrow. To exit the editor forward press shift and down arrow."